aries_cloudagent/protocols/present_proof/dif/proof_request.py

```diff
@@ -0,0 +1,29 @@
+"""DIF Proof Request Schema"""
+from marshmallow import fields, validate, validates_schema, ValidationError
+from ....messaging.models.openapi import OpenAPISchema
+from ....messaging.valid import UUID4
+from .pres_exch import PresentationDefinitionSchema
+
+
+class DIFProofRequestSchema(OpenAPISchema):
+    """Schema for DIF proof request."""
+
+    challenge = fields.String(
+        description="Challenge protect against replay attack",
+        required=False,
+        **UUID4,
+        data_key="challenge",
+    )
+    
+    domain = fields.String(
+        description="Domain protect against replay attack",
+        required=False,
+        data_key="domain",
+        example="4jt78h47fh47",
+    )   
+    
+    presentation_definition = fields.Nested(
+        PresentationDefinitionSchema(),
+        required=False,
+        data_key="presentation_definitions",
```

Comment:
We're going to change all occurrences of `definitions` to `definition` (singular instead of plural). Maybe we can do that right away?